### PR TITLE
Competition

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- Adds a short page describing out Full Stack Conference competition
- Includes Terms and Conditions

#### How can be this tested?
- Follow the directions to install the blog in the README.md
- `npm install`
- `npm start`
- visit http://127.0.0.1:4000/competition/full-stack-2015/

#### Screenshots / Screencast
screen shot 2015-08-04 at 20 43 27

#### What gif best describes how you feel about this work?
Golden Ticket
---
#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: competition/full-stack-2015.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: competition
+title: Full Stack Competition
+---
+
+![London]({{ site.url }}{{ site.baseurl }}/assets/img/london.png)
+
+To celebrate Holiday Extras sponsoring Full Stack Conference 2015, The Hungry Geek is pleased to offer you the chance to attend the conference in full (1 ticket), with accomodation for 3 nights in the St Giles hotel in the heart of London, just by entering our Twitter competition.
+
+All you have to do to enter is retweet a message from [Twitter](https://twitter.com/TheHungryGeek) about the competition and reply with why you want to go.
```
Comment (lukefrake) [inline competition/full-stack-2015.md]: Link to message?

Maybe mention you must be following HG to enter?

File: competition/full-stack-2015.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+The best answers will be placed into a draw and the winner will be announce at 5.30pm on Friday 25th September.
```
Comment (lukefrake) [inline competition/full-stack-2015.md]: announced?

File: competition/full-stack-2015.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+layout: competition
+title: Full Stack Competition
+---
+
+![London]({{ site.url }}{{ site.baseurl }}/assets/img/london.png)
+
+To celebrate Holiday Extras sponsoring Full Stack Conference 2015, The Hungry Geek is pleased to offer you the chance to attend the conference in full (1 ticket), with accomodation for 3 nights in the St Giles hotel in the heart of London, just by entering our Twitter competition.
```
Comment (notABluesSinger) [inline competition/full-stack-2015.md]: Spelling of accommodation

**Comment by notABluesSinger** [comment]
Taking Slot 1

**Comment by notABluesSinger** [comment]
:+1:

**Comment by HarleyRowland** [comment]
Taking second slot

**Comment by HarleyRowland** [comment]
+1

**Comment by lukefrake** [comment]
:shipit:

**Pull request merged**